l10n_it_delivery_note/readme/USAGE.rst

```diff
@@ -53,7 +53,7 @@ Di seguito, una piccola lista di passi da seguire per portare a termine la migra
 
 4. Lanciare Odoo, nella maniera in cui si è soliti fare, aggiungendo alcuni parametri al comando d’avvio:
 
-   `./odoo-bin migrate_ddt_data --database <nome_database> [...]`
+   `./odoo-bin migratel10nitddt --database <nome_database> [...]`
```

Comment:
Continuo a non essere d'accordo sul fatto che siano stati rimossi gli underscore dal nome della classe (e quindi, di conseguenza, dal nome del comando)...

Va bene seguire le regole di PyLint (o chicchessia) ma, personalmente parlando, in questo caso specifico reputo necessario aggiungere un'eccezione.  
Se il linter usato è sufficientemente figo, dovrebbe accettare una sorta di commento che specifichi di ignorare una determinata regola per la riga successiva.

Nella mia utopia, il comando dovrebbe essere qualcosa del tipo: `migrate_l10n_it_ddt`.